Source/Core/VideoBackends/Vulkan/VKShader.cpp

```diff
@@ -104,18 +104,21 @@ std::unique_ptr<VKShader> VKShader::CreateFromBinary(ShaderStage stage, const vo
                                                      size_t length)
 {
   ShaderCompiler::SPIRVCodeVector spv;
-  const size_t size_in_words = sizeof(length) / sizeof(ShaderCompiler::SPIRVCodeType);
+  const size_t size_in_words = length / sizeof(ShaderCompiler::SPIRVCodeType);
```

Comment:
This seems needlessly complicated to me -- most memcpy implementations will actually check for alignment already. Has this been profiled and measured to be required? If not, why not just a simple round-up + memcpy?